specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,435 @@
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            postcode:
+              $ref: '#/components/schemas/Postcode'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            land:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            datumAanvangAdreshouding:
+              $ref: 'datum.yaml#/components/schemas/GbaDatum'
+            datumAanvangAdresBuitenland:
+              $ref: 'datum.yaml#/components/schemas/GbaDatum'
+            gemeenteVanInschrijving:
+              $ref: '#/components/schemas/GemeenteVanInschrijving'
+            naamOpenbareRuimte:
+              $ref: '#/components/schemas/NaamOpenbareRuimte'
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    VerblijfplaatsBuitenlandInOnderzoek:
+      type: object
+      properties:
+        verblijfBuitenland:
+          type: boolean
+        datumVan:
+          type: boolean
+        datumIngangGeldigheid:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    VerblijfplaatsInOnderzoek:
+      type: object
+      properties:
+        aanduidingBijHuisnummer:
+          type: boolean
+        datumVan:
+          type: boolean
+        datumIngangGeldigheid:
+          type: boolean
+        datumInschrijvingInGemeente:
+          type: boolean
+        datumVestigingInNederland:
+          type: boolean
+        functieAdres:
+          type: boolean
+        gemeenteVanInschrijving:
+          type: boolean
+        huisletter:
+          type: boolean
+        huisnummer:
+          type: boolean
+        huisnummertoevoeging:
+          type: boolean
+        nummeraanduidingIdentificatie:
+          type: boolean
+        adresseerbaarObjectIdentificatie:
+          type: boolean
+        landVanwaarIngeschreven:
+          type: boolean
+        straat:
+          type: boolean
+        postcode:
+          type: boolean
+        korteNaam:
+          type: boolean
+        woonplaats:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    VerblijfplaatsOnbekendInOnderzoek:
+      type: object
+      properties:
+        land:
+          type: boolean
+    LocatieInOnderzoek:
+      type: object
+      properties:
+        datumInschrijvingInGemeente:
+          type: boolean
+        datumVestigingInNederland:
+          type: boolean
+        functieAdres:
+          type: boolean
+        gemeenteVanInschrijving:
+          type: boolean
+        landVanwaarIngeschreven:
+          type: boolean
+        locatiebeschrijving:
+          type: boolean
+        woonplaats:
```

Comment:
woonplaats hoeft hier niet in, want wordt bij een locatiebeschrijving altijd gevuld uit gemeenteVanInschrijving

Reply:
done